### DISCUSSION TITLE
How to perfomance check Image overrides for app?

### DISCUSSION THREAD

**Discussion by s4mur4i** [argoproj/argo-cd#5064] [Q&A] [answered]
Hello,

We are seeing about 5-12 seconds of wait for editing image tag overrides.
Checking developer mode I only see that a request is made upstream, waits and when call returns everything is okay.
I checked redis, don't see a bottleneck there. I also checked pods resource consumption, but no spike or further resources needed there.
Is there some way to see if bottleneck or improvement can be done our side? is it possible to profile where we have problem?
We are using argocd 1.7.4

Thank you

**Comment by alexmt** [comment, marked as answer]
The delay is happening because Argo CD tries to generate manifests to make sure that the introduced change is valid and won't "break" the application. The Argo CD CLI allows skipping the validation using `--validate=false` flag. This option is not yet available in UI.

**Reply by s4mur4i** [reply]
Thank you @alexmt for the response. After checking doc, one thought:
Would setting following options for example have same affect?
https://argoproj.github.io/argo-cd/user-guide/sync-options/#disable-kubectl-validation
Is there a plan to add it to UI?

**Reply by alexmt** [reply]
This request totally make sense. Can you please create an issue to add support `validate=false` in UI?

**Reply by s4mur4i** [reply]
Created an enhancement request https://github.com/argoproj/argo-cd/issues/5111
Thank you

**Comment by dirien** [comment]
Is it possible that the issue #5111 is already done? 

![image](https://user-images.githubusercontent.com/38325136/137026774-22b58783-fc64-4634-b51f-8f938fd89d25.png)
